### ISSUE TITLE
XRechnung 2.0.1 UBL Credit Note examples are broken

### ISSUE THREAD

**Issue by phax** [itplr-kosit/validator-configuration-xrechnung#51]
Hi @rkottmann, sorry to bug you again ;-)

Most of your UBL CreditNote test files from 2.0.1 trigger the error described in https://github.com/ConnectingEurope/eInvoicing-EN16931/issues/257 because of the duplicate `TaxTotal` element:

```xml
  <cac:TaxTotal>
      <cbc:TaxAmount currencyID="EUR">2048.44</cbc:TaxAmount>
      <cac:TaxSubtotal>
         <cbc:TaxableAmount currencyID="EUR">10781.25</cbc:TaxableAmount>
         <cbc:TaxAmount currencyID="EUR">2048.44</cbc:TaxAmount>
         <cac:TaxCategory>
            <cbc:ID>S</cbc:ID>
            <cbc:Percent>19</cbc:Percent>
            <cac:TaxScheme>
               <cbc:ID>VAT</cbc:ID>
            </cac:TaxScheme>
         </cac:TaxCategory>
      </cac:TaxSubtotal>    
  </cac:TaxTotal>
  
  
  <cac:TaxTotal>
      <cbc:TaxAmount currencyID="EUR">2048.44</cbc:TaxAmount>
  </cac:TaxTotal>
```

After removing
```xml
  <cac:TaxTotal>
      <cbc:TaxAmount currencyID="EUR">2048.44</cbc:TaxAmount>
  </cac:TaxTotal>
```
other (expected or unexpected) errors occur.
Please update your test file generator accordingly.

**Comment by rkottmann** [comment]
argh, forgot to inlcude them in the intgration tests..

@lkumai pls help out

**Comment by phax** [comment]
Fixed in EN release 1.3.5. Will be part of the next validation configuration release.
Key errror in this example: the 2 `<cac:TaxTotal>` elements need to use different currency IDs
